products/jbrowse-cli/test/data/trix/volvox2.json

```diff
@@ -0,0 +1,16 @@
+{
+  "TrackData": {
+    "TrackIds": [
+      "au9_scaffold",
```

Comment:
These tracks e.g. au9_scaffold do not belong to volvox2 assembly, why are they in volvox2.json?